test_settings.py

```diff
@@ -90,7 +93,7 @@
             },
             {
                 'code': 'fr',
-                'name': 'Française',
+                'name': u'Française',
```

Comment:
Please remove this. `from __future__ import unicode_literals` is in effect, so, the 'u' is not required nor desired.